text/2996-async-iterator.md

```diff
@@ -1,69 +1,70 @@
-- Feature Name: `async_stream`
+- Feature Name: `async_iterator`
 - Start Date: 2020-09-29
 - RFC PR: [rust-lang/rfcs#2996](https://github.com/rust-lang/rfcs/pull/2996)
 - Rust Issue: [rust-lang/rust#79024](https://github.com/rust-lang/rust/issues/79024)
 
 # Summary
 [summary]: #summary
 
-Introduce the `Stream` trait into the standard library, using the
+Introduce the `AsyncIterator` trait into the standard library, using the
 design from `futures`. Redirect the `Stream` trait definition in the 
-`futures-core` crate (which is "pub-used" by the `futures` crate) to the standard library.
+`futures-core` crate (which is "pub-used" by the `futures` crate) to the
+`AsyncIterator` trait in the standard library.
 
 # Motivation
 [motivation]: #motivation
 
-Streams are a core async abstraction. These behave similarly to `Iterator`,
+Async iterators are a core async abstraction. These behave similarly to `Iterator`,
 but rather than blocking between each item yield, it allows other
 tasks to run while it waits.
 
 People can do this currently using the `Stream` trait defined in the 
 [futures](https://crates.io/crates/futures) crate. However, we would like
-to add `Stream` to the standard library. 
+to add `AsyncIterator` to the standard library as `AsyncIterator`. 
```

Comment:
to add `Stream` to the standard library as `AsyncIterator`?

Reply:
good call; updated!